konrad/cloud.py

```diff
@@ -138,21 +224,77 @@ def update_cloud_profile(self, *args, **kwargs):
         return
 
 
+class DirectInputCloud(Cloud):
```

Comment:
I am still not fully convinced that this will make life easier for the user, but we can try 😉

Reply:
I _hope_ it makes it easier!